src/components/AvatarWithImagePicker.js

```diff
@@ -116,112 +114,115 @@ const defaultProps = {
     originalFileName: '',
 };
 
-class AvatarWithImagePicker extends React.Component {
-    constructor(props) {
-        super(props);
-        this.animation = new SpinningIndicatorAnimation();
-        this.setError = this.setError.bind(this);
-        this.isValidSize = this.isValidSize.bind(this);
-        this.showAvatarCropModal = this.showAvatarCropModal.bind(this);
-        this.hideAvatarCropModal = this.hideAvatarCropModal.bind(this);
-        this.state = {
-            isMenuVisible: false,
-            validationError: null,
-            phraseParam: {},
-            isAvatarCropModalOpen: false,
-            imageName: '',
-            imageUri: '',
-            imageType: '',
-        };
-        this.anchorRef = React.createRef();
-    }
-
-    componentDidMount() {
-        if (!this.props.isUploading) {
-            return;
+function AvatarWithImagePicker({ 
+    isUploading, 
```

Comment:
<img width="550" alt="image" src="https://github.com/Expensify/App/assets/9059945/99b3b57c-52b5-41e8-aa3c-4b699b3fbeb9">

Reply:
We're not using `compose` anymore, so the same linter warning

<img width="732" alt="image" src="https://github.com/Expensify/App/assets/9059945/6cb6aef3-5c8a-411a-9909-26e39c6e32d6">